.github/workflows/node.js.yml

```diff
@@ -9,12 +9,32 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install
-        run: yarn --ci
-      - name: Test
-        run: yarn test
-      - name: Build
-        run: yarn build
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+
+      - name: Tag Build
+        uses: docker/metadata-action@v3
+        id: meta
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=ref,event=pr
+            type=ref,event=branch
+            type=sha
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}
```

Comment:
Currently unused, but this is a PoC for future changes. It does in fact work! https://github.com/reactiflux/reactibot/pkgs/container/reactibot